### PR TITLE
Fixed some typos and minor grammatical errors

### PR DESCRIPTION
**Pull request by DCzajkowski**
- [consisting of, not on](https://www.dictionary.com/browse/consisting),
- a UI, not an UI, as articles are [based on pronunciation](https://www.a-or-an.com/a_an/ui),
- an SQL, as articles are based on pronunciation, and SQL is pronounced `es-q-el`,
- look at, not look on,
- ["login" is a noun, "to log in" is a verb](http://grammarist.com/spelling/log-in-login/)
- [commas in the if statement](https://www.ego4u.com/en/cram-up/writing/comma?10)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,18 +3,18 @@
 
 # Source Code Annotation Tool
 
-Training Machine Learning models often requires large datasets to be duly annotated.
+Training Machine Learning models often require large datasets to be duly annotated.
```
Comment (carlosms) [inline README.md]: I believe it's _requires_.

_Training Machine Learning models_ often requires
_It_ often requires

File: README.md [hunk 2]
```diff
@@ -131,7 +131,7 @@ The annotations made by the users will be stored in the **`assignments`** table.
 
 It is possible to restrict access and choose each user's role by adding their GitHub accounts to a specific [organization](https://help.github.com/articles/collaborating-with-groups-in-organizations/) or [team](https://help.github.com/articles/organizing-members-into-teams/).
 
-This is optional, if you don't set any restrictions all users with a valid GitHub account will be able to login as a Requester. You may also set a restriction only for Requester users, and leave open access to anyone as Workers.
+This is optional if you don't set any restrictions all users with a valid GitHub account will be able to log in as a Requester. You may also set a restriction only for Requester users, and leave open access to anyone as Workers.
```
Comment (carlosms) [inline README.md]: I believe the comma is needed to separate the sentences. "if you don't set any restrictions" is not a condition for this to be optional.

**Review by carlosms — CHANGES_REQUESTED** [review]
Thanks!

I left a couple of comments.

File: README.md (hunk 2 above)
Comment (DCzajkowski) [reply README.md]: What about:
- "This is optional, but if you don't set any restrictions, all users with a valid GitHub account will be able to log in as a Requester."
- "This is optional. If you don't set any restrictions, all users with a valid GitHub account will be able to log in as a Requester."

File: README.md (hunk 1 above)
Comment (DCzajkowski) [reply README.md]: You are right. My bad, sorry.

**Comment by DCzajkowski** [comment]
Let me know if it's alright now :)

**Review by dpordomingo — APPROVED** [review]

**Review by carlosms — APPROVED** [review]
Great, thank you

**Pull request merged**